docs/how_to_config.md

```diff
@@ -57,22 +57,69 @@ bit more "Pythonic" and easier to read.)
 This is a template for `.pre-commit-config.yaml`:
 
 ```yaml
+# Option 1: with native tool
```

Comment:
Can we do this?  This looks better and can save some page space.

<table>
  <tr>
    <td>
    As a native tool
    </td>
    <td>
    As a flake8 plugin
    </td>
  </tr>
  <tr>
    <td>

```yaml
- repo: https://github.com/jsh9/pydoclint
  rev: <latest_tag>
  hooks:
    - id: pydoclint
      args:
        - --style=google
        - --check-return-types=False
```
  </td>

  <td>

```yaml
- repo: https://github.com/pycqa/flake8
  rev: <latest_tag>
  hooks:
    - id: flake8
      additional_dependencies:
        - pydoclint==<latest_tag>
      args:
        - --style=google
        - --check-return-types=False
```
  </td>

  </tr>
</table>

Reply:
Ooo this is cool! I will add this sometime this weekend

Reply:
Implemented, about to push